### PR TITLE
Adds `estimated_total` cols for `graphql_ahr`

### PR DESCRIPTION
**Pull request by benhammondmusic**
# Description and Motivation

- part of #3360 
- uses new generate estimated total util fn to estimate number of suicides using full pop.; and numbers of others (except voting and prev. hosp) based on the 18+ pop. NOTE: we can probably figure out getting these for the by age table later, but skipped for now as it's untested. 
- adds new helper function to extract topic prefixes from metric column names
- cleanup some typing, linting, and use of constants
- updates golden data to expect new cols and new data

## Has this been tested? How?

Tests updated and passing 

## Types of changes

(leave all that apply)

- New content or feature

### CODE REVIEW

File: python/tests/datasources/test_graphql_ahr.py [hunk 1]
```diff
@@ -1,11 +1,10 @@
...
-
-from datasources.graphql_ahr import GraphQlAHRData
-from ingestion.constants import AGE, CURRENT, NATIONAL_LEVEL, SEX, STATE_LEVEL
-from ingestion.standardized_columns import RACE_OR_HISPANIC_COL, STATE_FIPS_COL
-from pandas._testing import assert_frame_equal
+import pandas as pd  # type: ignore
+from pandas._testing import assert_frame_equal  # type: ignore
+from datasources.graphql_ahr import GraphQlAHRData  # type: ignore
+from ingestion.constants import AGE, CURRENT, NATIONAL_LEVEL, SEX, STATE_LEVEL  # type: ignore
+from ingestion.standardized_columns import RACE_OR_HISPANIC_COL, STATE_FIPS_COL  # type: ignore
```
Comment (eriwarr) [inline python/tests/datasources/test_graphql_ahr.py:7]: Why `# type: ignore`?

File: python/ingestion/standardized_columns.py [hunk 2]
```diff
@@ -421,3 +435,15 @@ def generate_column_name(prefix, suffix):
     suffix: a type of measurement (pct_share, per_100k)"""
 
     return f'{prefix}_{suffix}'
+
+
+def extract_prefix(col_name: str) -> str:
+    """Extracts the prefix from a column name that contains one of out standard HET suffixes."""
```
Comment (eriwarr) [inline python/ingestion/standardized_columns.py:441]: Love this!

File: python/ingestion/graphql_ahr_utils.py [hunk 3]
```diff
@@ -33,11 +32,30 @@ def generate_cols_map(prefixes: list[str], suffix: str):
     'Diabetes': 'diabetes_per_100k',
     'Excessive Drinking': 'excessive_drinking_per_100k',
     'Frequent Mental Distress': 'frequent_mental_distress_per_100k',
-    'Preventable Hospitalizations': 'preventable_hospitalizations_per_100k',
     'Non-Medical Drug Use - Past Year': 'non_medical_drug_use_per_100k',
+}
+
+AHR_MEASURES_TO_RATES_MAP_ALL_AGES = {
```
Comment (eriwarr) [inline python/ingestion/graphql_ahr_utils.py:38]: I think that the suicide measure is ages 15+

**Review by eriwarr — APPROVED** [review]

File: python/tests/datasources/test_graphql_ahr.py (hunk 1 above)
Comment (benhammondmusic) [reply python/tests/datasources/test_graphql_ahr.py:7]: i was lazy and my local python vscode extension running mypy was angry. i think i fixed it better though by adding settings to the pyproject.toml

File: python/ingestion/graphql_ahr_utils.py (hunk 3 above)
Comment (benhammondmusic) [reply python/ingestion/graphql_ahr_utils.py:38]: i can't figure out this for sure

**Pull request merged**